app/components/utils/edit-mandatee/component.js

```diff
@@ -36,29 +55,37 @@ export default Component.extend({
     },
 
     async chooseDomain(domains) {
-      this.set('selectedDomains', domains)
+      this.set('selectedDomains', domains);
       this.chooseDomain(domains);
     },
 
     personSelected(person) {
       this.set('mandateeToEdit.person', person);
     },
 
-    async saveChanges() {
-      this.set('isLoading', true);
-      const { startDate, endDate, title, shortTitle, mandateeToEdit, iseCodes, nickName } = this;
-      const mandatee = await this.store.findRecord('mandatee', mandateeToEdit.get('id'));
-      mandatee.set('end', endDate);
-      mandatee.set('title', title);
-      mandatee.set('shortTitle', shortTitle);
-      mandatee.set('iseCodes', iseCodes);
-      mandatee.set('start', startDate);
-      mandatee.set('nickName', nickName);
-      mandatee.save().then(() => {
-        this.set('isLoading', false);
-        this.closeModal();
-        this.mandateesUpdated();
-      });
+    verify() {
+      this.showVerificationPopup = !this.showVerificationPopup;
+    },
+
+    cancel() {
+      this.showVerificationPopup = !this.showVerificationPopup;
```

Comment:
This is actually a toggle. If you name this function cancel, I would explicitly set `this.showVerificationPopup` to `false`

Reply:
Done